Description: Switch wheel to netdev group in dbus policy
 In Debian there's a specific dedicated group
 that is supposed to give permission to manage the network,
 called netdev.
 Thus replace the generic,
 and in debian not very widely used,
 wheel group in the default policy.
Author: Andreas Henriksson <andreas@fatal.se>
Last-Update: 2020-10-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/iwd-dbus.conf
+++ b/src/iwd-dbus.conf
@@ -14,7 +14,7 @@
     <allow send_interface="net.connman.iwd.SignalLevelAgent"/>
   </policy>
 
-  <policy group="wheel">
+  <policy group="netdev">
     <allow send_destination="net.connman.iwd"/>
   </policy>
 
